projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -2842,8 +2842,13 @@ function process_submission() {
 			$reply_to_addr = $comment_author_email;
 		}
 
-		$headers = 'From: "' . $comment_author . '" <' . $from_email_addr . ">\r\n" .
-		           'Reply-To: "' . $comment_author . '" <' . $reply_to_addr . ">\r\n";
+		$headers = sprintf(
+			'From: %1$s <%2$s>%4$sReply-To: %1$s <%3$s>%4$s',
+			$comment_author,
+			$from_email_addr,
+			$reply_to_addr,
+			"\r\n"
+		);
```

Comment:
I had to do more than just remove the quotes here to avoid phpcs warnings on the indenting, that would stop me from committing.